### ISSUE TITLE
Schweizer Adressauflösung und -eingabe

### ISSUE THREAD

**Issue by innosmith** [decidim-swiss/decidim.swiss#14]
Auf deinKlima INT kamen nur Luzerner Adressen als gültige Adressen zur Auswahl. In der Folge habe ich im Systemdashboard Start- und Endkoordinaten vermeintlich auf die ganze Schweiz eingestellt. Ergebnis deutsche Adressen ;-) Bitte korrigieren und sicherstellen, dass auf INT und PROD von deinKlima bei der Adresseingabe die ganze Schweiz abgedeckt ist. Der Startpunkt darf gerne in Bern sein, wo die Wyss Academy und Kt. Bern zu Hause sind. Wie hiess noch einmal das Koordinaten-Tool?

**Comment by carlobeltrame** [comment]
Koordinator: https://tools.retorte.ch/map

**Comment by larsUE** [comment]
Auf gemeinsamer.space hat es geklappt :-) Allerdings tritt auch da der [Zürcher Kartenbug](https://github.com/puzzle/decidim-zuerich/issues/267) auf

**Comment by carlobeltrame** [comment]
Habe das Zentrum in Bern von deinklima INT zu PROD übernommen, und das Koordinaten-Rechteck von gemeinsamer.space auf deinklima INT und PROD übernommen. Sollte nun alles konsistent sein.

**Comment by carlobeltrame** [comment]
Übrigens ist es natürlich immer noch möglich dass Adressen in Deutschland gefunden werden. Die Schweiz ist ja schliesslich kein Rechteck. Aber die Orte sollten nicht weit von der Schweiz entfernt sein.

**Comment by innosmith** [comment]
Bestens, vielen Dank @carlobeltrame . Nun benötigen wir "nur" noch den Bugfix aus Zürich ;-)

**Comment by innosmith** [comment]
@Kagemaru hast du hierzu allenfalls schon Fortschritte erzielt, welche wir vor dem morgigen Deployment auch für deinKlima und Dialog Luzern anwenden könnten?

**Comment by Kagemaru** [comment]
Nein, leider bin ich bis jetzt nicht auf einen Grund gestossen, warum dies passiert.

**Comment by larsUE** [comment]
Schaut im Zürcher Ticket, ich glaube ich konnte den exakten Fehler nun Nachreproduzieren.

**Comment by larsUE** [comment]
Ich habe den Fix von Carlo ausführlich auf gemeinsamer.space getestet und es funktioniert wie gewünscht.
